.github/actions/release-plugin/action.yml

```diff
@@ -77,6 +82,38 @@ runs:
       with:
         token: ${{ inputs.github_token }}
         files: ${{ env.org-zip-path }}
-        asset_name: faustwp-${{ env.VERSION }}.org.zip
-        tag_name: ${{ env.VERSION }}
+        asset_name: faustwp.${{ env.VERSION }}.org.zip
+        tag_name: canary
+        overwrite: true
+
+    - id: generate-info-json
+      name: Fetch and update info.json for WPE release
+      run: |
+        # Fetch the info.json file and set variables
```

Comment:
Should this be a script instead? we want to run shellcheck as well and its not easy to verify it here

Reply:
![Uploading Screenshot 2025-03-06 at 09.19.19.png…]()

Reply:
Thanks @theodesp 

I have now extracted it out to a bash script and tested in my previous commit :)